app/views/attachments/_form.html.erb

```diff
@@ -1,30 +1,40 @@
 <%= form_for @attachment, as: :attachment, url: path, builder: FormBuilderWithDateTimeInput do |f| %>
   <%= f.error_messages %>
 
-	<%= f.text_field :name, :required => true %>
-
-	<% if @attachment.new_record? %>
-  <div class="row embedded-form">
-    <label class="custom-file-upload">
-			<span class="btn primary" id="upload_button">
-				Attachment ...
-					<%= f.file_field :file, class: "upload_embedded" %>
-			</span>
-		</label>
-		<input type="text" id="upload_file" readonly value="Select to Upload a File.">
-  </div>
-	<% else %>
-		<%= f.text_field :mime_type,
-				help_text: "Note: If you change the Mime type of a file, you might need to clear your browser's cache in order for you to see the change." %>
-	<% end %>
+	<ul>
```

Comment:
Is the purpose of `<ul>` to make the lines the same length?

Reply:
Pretty much